src/Umbraco.Web.BackOffice/Filters/MemberSaveModelValidator.cs

```diff
@@ -23,6 +28,24 @@ internal class
     private readonly IMemberService _memberService;
     private readonly IMemberTypeService _memberTypeService;
     private readonly IShortStringHelper _shortStringHelper;
+    private readonly SecuritySettings _securitySettings;
+
+    public MemberSaveModelValidator(
+        ILogger<MemberSaveModelValidator> logger,
+        IBackOfficeSecurity? backofficeSecurity,
+        IMemberTypeService memberTypeService,
+        IMemberService memberService,
+        IShortStringHelper shortStringHelper,
+        IPropertyValidationService propertyValidationService,
+        SecuritySettings securitySettings)
+        : base(logger, propertyValidationService)
+    {
+        _backofficeSecurity = backofficeSecurity;
+        _memberTypeService = memberTypeService ?? throw new ArgumentNullException(nameof(memberTypeService));
+        _memberService = memberService ?? throw new ArgumentNullException(nameof(memberService));
+        _shortStringHelper = shortStringHelper ?? throw new ArgumentNullException(nameof(shortStringHelper));
+        _securitySettings = securitySettings;
+    }
 
     public MemberSaveModelValidator(
```

Comment:
You can remove this constructor, because it's not used and `MemberSaveModelValidator` is an internal class 😄

If it was a public class however, it should me marked as obsolete and call the new constructor using `this(...)`, so the fields (and null checks) don't have to be duplicated.